journal/modernized/21-chapter-18.adoc

```diff
@@ -326,8 +327,8 @@ There were two men not far off, who belonged to them, but they kept aloof.
 These women appeared to be in a poor state of health, and exceedingly shrunk:
 they could talk a little English, and on our making them some trifling presents,
 and saying that some of us would see them again, if they should be there tomorrow,
-one of them said, 'You welcome;
-come.' It was affecting to behold the degraded condition of these natives of the soil.
+one of them said, "`You're welcome;
+come.`" It was affecting to behold the degraded condition of these natives of the soil.
```

Comment:
"You welcome" is in the pdf too. It doesn't seem right to keep it that way so I changed it.

Reply:
I think this is a case where Wheeler is directly quoting broken English, as he says above "they could talk a little English".  There are some other places later in the book where he puts in word for word stumbling attempts to speak English or poor translations into English, which gives me the confidence to say that I think this is why it is the way it is.  So I think we should probably switch this back. But I _totally_ understand why you changed it, it is a bit confusing.

Reply:
Ahhh that makes sense. I should have noticed that. I don't think I read far enough up.